app/models/comment.rb

```diff
@@ -263,25 +263,22 @@ def notify_parent_comment_owner
         # send notification to the owner of the original comment if they're not the same as the commenter
         if (have_different_owner?(parent_comment))
           if !parent_comment_owner || notify_user_by_email?(parent_comment_owner) || self.ultimate_parent.is_a?(Tag)
-            if self.edited_at_changed?
+            if self.saved_change_to_edited_at?
               CommentMailer.edited_comment_reply_notification(parent_comment, self).deliver
             else
               CommentMailer.comment_reply_notification(parent_comment, self).deliver
             end
           end
           if parent_comment_owner && notify_user_by_inbox?(parent_comment_owner)
-            if self.edited_at_changed?
+            if self.saved_change_to_edited_at?
               update_feedback_in_inbox(parent_comment_owner)
             else
               add_feedback_to_inbox(parent_comment_owner)
             end
           end
-          if parent_comment_owner
-            return parent_comment_owner
-          end
```

Comment:
I'm a little wary that might break a test - I honestly can't think of why I would have removed these lines otherwise. But all of the tests I could think of to run associated with comments passed locally so perhaps it was just an odd oversight.